contributing/CHANGES.md

```diff
@@ -1,8 +1,8 @@
-# Recording user facing changes in CHANGES.md
```

Comment:
[fix] The example under "Formatting" below still has "Changes" in the title (maybe the title is irrelevant to that section and can be removed)

Reply:
Good spot, thanks! Fixed [here](https://github.com/OpenAssetIO/OpenAssetIO/pull/386/commits/0a9f7bf51e7070e80142554ac20bd7f81d9c4515).